tests/dynamic_data/base_test.py

```diff
@@ -111,7 +110,7 @@ def parametrize_test_cases(cls) -> [TestCaseType]:
                 if input_publication:
                     name = input_publication.name or name
 
-                rest_method = parametrization.rest_method or RestMethod.POST
+                rest_method = parametrization.rest_method or input_test_case.rest_method or RestMethod.POST
```

Comment:
Add assert `assert input_test_case.rest_method is None or parametrization.rest_method is None` somewhere to the cycle

Reply:
+ test would be nice